src/Definition/GraphQLServices.php

```diff
@@ -81,4 +82,12 @@ public function getType(string $typeName): ?Type
     {
         return $this->types->resolve($typeName);
     }
+
+    /**
+     * Creates an instance of InputValidator
+     */
+    public function createInputValidator(array $resolverArgs): InputValidator
```

Comment:
Can we decompose resolve args to have all types definitions?

Reply:
done